test/unittests/mongoBackend/mongoContextProvidersUpdate_test.cpp

```diff
@@ -523,20 +523,27 @@ TEST(mongoContextProvidersUpdateRequest, noPatternAttrOneSingle)
   EXPECT_EQ(SccOk, ms);
 
   EXPECT_EQ(SccNone, res.errorCode.code);
```

Comment:
We need to talk about this with SccNone again ...
Here, clearly mongoUpdateContext doesn't touch the errorCode.
The way I have understood your idea this is an *error* and should be treated as such.
If errorCode is initialized to SccNone, everything works and after the call to mongoUpdateContext, errorCode.code is still SccNone ... what can I do in the upper layer?

If we are to continue with SccNone, **all** mongoBackend functions must fill it in with either SccOk or with some error code. If not, how is the upper layer to know whether the function worked or not? [ it always returns OK ].

The way it is right now, the only way to use it is to consider SccNone == SccOk, or filling in the errorCode.code to SccOk before calling mongoUpdateContext (which is the same).

This has been giving me a lot of problems lately.

Only two options:
1. Initialize to SccNone and mongoBackend ALWAYS puts it to either SccOk or some error.
2. Initialize to SccOk and mongoBackend ONLY touches it in case of error.

Right now we are in the invalid option:
3. Initialize to SccNone and mongoBackend SOMETIMES puts it to either SccOk or some error.